x/wasm/types/types.go

```diff
@@ -429,3 +430,29 @@ func (a AccessConfig) AllAuthorizedAddresses() []string {
 	}
 	return []string{}
 }
+
+type txContracts map[string]struct{}
+
+type TxContracts struct {
+	contracts txContracts
+}
```

Comment:
Nice! Could you add unit tests for `TxContracts` to ensure the existence checks are working